src/apm/php.go

```diff
@@ -78,14 +93,14 @@ func (i *PhpInjector) Inject(ctx context.Context, inst v1alpha2.Instrumentation,
 
 	firstContainer := 0
 
-	// exit early if we're missing mandatory annotations
-	// Deprecated: phpVer is deprecated.  Do not use annotations.
-	phpVer, ok := pod.Annotations[annotationPhpVersion]
-	if !ok {
-		return pod, errors.New("missing php version annotation")
+	// <lang:php>-<version:[0-9].[0-9]>
+	lang := strings.SplitN(i.Language(), "-", 2)
+	if len(lang) != 2 {
+		// should never happen
+		return pod, errors.New("missing php version")
 	}
+	phpVer := lang[1]
```

Comment:
Rather than splitting the string to try and parse out the PHP version number, I would suggest just updating the key values in the current [api map](https://github.com/newrelic/k8s-agents-operator/pull/104/files#diff-9d8393bab101cb85655446396e995080f844370d607c6b39ae4e560f684e138aR42) to be able to access the ZEND API Number directly.

Reply:
will fix in next update